_site/index.html

```diff
@@ -510,6 +510,20 @@ <h3>T-Z</h3>
           <p class="word__definition">
             the fear, hatred, and stigmitazation of fat people.<br><br>
           </p>
+          <div class="block__type">
+            <p class="word__type">
+              Issues
+            </p>
+            <p class="word__breakdown">It perpetuates the idea that there is a “normal” body type, that there is a direct correlation between thinness and heathliness (which also perpetuates healthiness as an indicator of value), and that fatness is both
```

Comment:
Do you feel comfortable adding factors around class and race?

Typically fatphobia exacerbates racialisation of what an "ideal" body type is as well through a white and Western lens.

Reply:
Yes, absolutely - those are important factors, thank you for catching that.